guidelines/index.html

```diff
@@ -1586,23 +1586,23 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 	<p class="change">[New]</p>
     
-    <p>The size of the <a href="#dfn-targets" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-targets-2" title="ユーザインタフェース コンポーネントのインタラクティブな領域のような、ポインタアクションを受け入れるディスプレイの領域">target</a> for <a href="#dfn-pointer-inputs" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-pointer-inputs-2" title="マウス、ペン、タッチ接触のように、画面上の特定の座標 (又は複数の座標群) をターゲットにできるデバイスからの入力。">pointer inputs</a> is at least 24 by 24 <a href="#dfn-css-pixels" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-css-pixels-5" title="約 0.0213 度の視野角。">CSS pixels</a>, except where:</p>
+    <p><a href="#dfn-pointer-inputs" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-pointer-inputs-2" title="マウス、ペン、タッチ接触のように、画面上の特定の座標 (又は複数の座標群) をターゲットにできるデバイスからの入力。">ポインタ入力</a>の<a href="#dfn-targets" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-targets-2" title="ユーザインタフェース コンポーネントのインタラクティブな領域のような、ポインタアクションを受け入れるディスプレイの領域">ターゲット</a>のサイズが 24 × 24 <a href="#dfn-css-pixels" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-css-pixels-5" title="約 0.0213 度の視野角。">CSS ピクセル</a>以上である。ただし、以下の場合は例外とする。</p>
     <ul>
-        <li><strong>Spacing:</strong> Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the <a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-1" title="[New]">bounding box</a> of each, the circles do not intersect another target or the circle for another undersized target;</li>
-        <li><strong>Equivalent:</strong> The function can be achieved through a different control on the same page that meets this criterion;</li>
-        <li><strong>Inline:</strong> The target is in a sentence or its size is otherwise constrained by the line-height of non-target text;</li>
-        <li><strong>User agent control:</strong> The size of the target is determined by the <a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-3" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">user agent</a> and is not modified by the author;</li>
-        <li><strong>Essential:</strong> A particular <a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-4" title="rendering of the content in a form to be perceived by users">presentation</a> of the target is <a href="#dfn-essential" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-essential-15" title="もし取り除いてしまうと、コンテンツの情報又は機能を根本的に変えてしまい、かつ、適合する他の方法では情報及び機能を実現できない。">essential</a> or is legally required for the information being conveyed.</li>
+        <li><strong>スペースがある:</strong> 小サイズ (24 × 24 CSS ピクセル未満) のターゲットにおいて、各ターゲットの<a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-1" title="[New]">バウンディングボックス</a>の中心に直径 24 CSS ピクセルの円を置いた場合、その円が別のターゲットもしくは別の小サイズのターゲットの円と交差しないように、ターゲットが配置されている。</li>
```

Comment:
土屋さん訳のように例外で「～がある」としてもらうのがよいのか、あるいは体言止めのほうがよいのかが謎ではありますが…ひとまず体言止めの方向に寄せてみます。
ここでは互いの間隔を取るということかと思いますので、「間隔」としてみました。

```suggestion
        <li><strong>間隔:</strong> 小さなターゲット (24 × 24 CSS ピクセル未満のもの) は、直径 24 CSS ピクセルの円をそれぞれの<a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-1" title="[New]">境界ボックス</a>の中心に置く場合、その円が別のターゲット又は別の小さなターゲットの円と交差しないように配置されている。</li>
```

Reply:
項目名を体言止めにするかは、既存の達成基準 2.5.5 と併せて確認したいと思います。

それ以外の修正提案は、異論ありません。